ongebo/fff-frontend
# #163673497 Initialise repository as a react project

### What does this PR do?
Sets up the repository as a React project
#### Description of the task to be completed?
- Create README.md, .gitignore and an MIT license file
- Initialise repo as a yarn project with package.json and yarn.lock
- Create public folder to hold index.html and other static assets
- Install Babel and add its configuration to .babelrc
- Install webpack dependencies for bundling the application
- Add webpack configuration to webpack.config.js
- Install ESLint dependencies with configuration in .eslintrc.json
- Install React and ReactDOM
- Create basic React app files in the src folder
#### How should this be manually tested?
- Install all dependencies by running `yarn`
- Start the development server using the command `yarn start`
- Open `localhost:3000` in the browser to verify that the application is running
#### Screenshot
![image](https://user-images.githubusercontent.com/34346070/52168321-b41e1400-2739-11e9-8962-30b3f227a2e0.png)
#### What are the relevant Pivotal Tracker stories?
[#163673497](https://www.pivotaltracker.com/story/show/163673497)
